app/cells/plugins/core/user_cell.rb

```diff
@@ -14,7 +14,7 @@ def value
       end
 
       def render_select
-        @options[:form].select 'data[user_id]', user_data_for_select, {selected: value}
+        @options[:form].select 'data[user_id]', user_data_for_select, {selected: value}, required: required?
       end
 
       def user_data_for_select
```

Comment:
I assume adding `required` to fields like `file` didn't do anything? Just checking - we said we'd try (and drop it quickly), so just making sure that was looked into